### PR TITLE
Update roadmap.md

### PR DESCRIPTION
**Pull request by petermarks-gsa**
Updated milestone 2

### CODE REVIEW

File: docs/product/roadmap.md [hunk 1]
```diff
@@ -29,33 +29,43 @@ Additional value here is preventing entities that don't meet the criteria from s
...
 **Value delivered:**
 To de-risk our product plans, we will focus on an end-to-end solution that bridges data input to dissemination to the current FAC. This will include a fully fleshed out SF-SAC general information page. We will also build out the API to de-risk unknows in end-to-end implementation and unblock our partners at Census by providing specifics of API format. 
 
+Submit general information end to end:
 * (20% ✅) [General information - SF-SAC data entry](https://github.com/GSA-TTS/FAC/issues/185)
-* [API to transfer data to Census](https://github.com/GSA-TTS/FAC/issues/310)
-* [PDF Validation Spike](https://github.com/GSA-TTS/FAC/issues/307)
-* (We've already done a spike around Excel file ingestion)
-* (60% ✅) design/exploration - "Submit an entire audit package" milestone
-* (design/exploration) Status and user business logic
```
Comment (MelissaBraxton) [inline docs/product/roadmap.md:46]: I'd say this is already 50% on the design side, but nbd.

File: docs/product/roadmap.md [hunk 2]
```diff
@@ -29,33 +29,43 @@ Additional value here is preventing entities that don't meet the criteria from s
...
 
 **Value delivered:**
 To de-risk our product plans, we will focus on an end-to-end solution that bridges data input to dissemination to the current FAC. This will include a fully fleshed out SF-SAC general information page. We will also build out the API to de-risk unknows in end-to-end implementation and unblock our partners at Census by providing specifics of API format. 
 
+Submit general information end to end:
 * (20% ✅) [General information - SF-SAC data entry](https://github.com/GSA-TTS/FAC/issues/185)
-* [API to transfer data to Census](https://github.com/GSA-TTS/FAC/issues/310)
-* [PDF Validation Spike](https://github.com/GSA-TTS/FAC/issues/307)
-* (We've already done a spike around Excel file ingestion)
-* (60% ✅) design/exploration - "Submit an entire audit package" milestone
```
Comment (MelissaBraxton) [inline docs/product/roadmap.md:45]: This is very close to done, but is dependent on reopening the Q of whether we'll be accepting data vis web form as well as Excel in the initial release.

**Review by MelissaBraxton — APPROVED** [review]
Added a few optional comments, but it looks good.

File: docs/product/roadmap.md (hunk 2 above)
Comment (petermarks-gsa) [reply docs/product/roadmap.md:45]: Updated

File: docs/product/roadmap.md (hunk 1 above)
Comment (petermarks-gsa) [reply docs/product/roadmap.md:46]: Updated

File: docs/product/roadmap.md [hunk 3]
```diff
@@ -29,33 +29,43 @@ Additional value here is preventing entities that don't meet the criteria from s
...
 
 **Specific deliverables:**
-* PDF Validation (text, searchable, unlocked)
-* Excel file ingest 
+* Certify audit submission
+* PDF Validation
+* Excel file ingest
 * Submit an audit package (post SAC forms)
 * (design/exploration) Status and user business logic
+* [PDR - Audit processing](https://github.com/GSA-TTS/FAC/issues/329)
```
[the remark this answers is not in the document]
Comment (petermarks-gsa) [reply docs/product/roadmap.md]: Renamed to "Automated audit processing"

**Pull request merged**